commands/flags.go

```diff
@@ -69,20 +69,22 @@ type FlagsStruct struct {
 	}
 
 	property struct {
-		cert          bool
-		key           bool
-		auth          bool
-		apihost       bool
-		apiversion    bool
-		namespace     bool
-		cliversion    bool
-		apibuild      bool
-		apibuildno    bool
-		insecure      bool
-		all           bool
-		apihostSet    string
-		apiversionSet string
-		namespaceSet  string
+		cert              bool
+		key               bool
+		auth              bool
+		apihost           bool
+		apiversion        bool
+		namespace         bool
+		promptOnChange    bool
+		cliversion        bool
+		apibuild          bool
+		apibuildno        bool
+		insecure          bool
+		all               bool
+		apihostSet        string
+		apiversionSet     string
+		namespaceSet      string
+		promptOnChangeSet string
```

Comment:
can this be a boolean flag?  i.e. the present of `--confirm` (typical practice in other clis use `--force`) indicates the desire to override the confirmation.

Reply:
Already modified